templates/LDA_Preprocessing.ipynb

```diff
@@ -1,15 +1,941 @@
 {
```

Comment:
<p>This is useful for exploratory data analysis, but I think we can assume that people will have correctly formatted cell tables, since they will have been produced by a previous step in the pipeline. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/ark-analysis/pull/468/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>